src/System/Logging/LogEventArgs.cs

```diff
@@ -24,6 +25,7 @@ public LogEventArgs(uint tagid, Category category, Level level, string message,
 			Category = category;
 			Level = level;
 			TagId = tagid;
+			ThreadId = Thread.CurrentThread.ManagedThreadId;
```

Comment:
>CurrentThread [](start = 21, length = 13)

While this should never be null, I'd suggest being defensive (Thread.CurrentThread?.ManagedThreadId)